tests/test_s3.py

```diff
@@ -126,11 +195,17 @@ async def test_flow():
     )
 
 
-async def test_s3_list_objects_prefix(object, object_in_folder, aws_credentials):
+@pytest.mark.parametrize("s3_mock, client_parameters", aws_clients, indirect=True)
```

Comment:
I don't think that `s3_mock` needs to be parameterized with your recent addition of the `client_parameters` fixture. Removing the `s3_mock` parameterization would me that you don't need a list of tuples for `aws_clients`.

Reply:
Indeed, you are right. I update the fixtures usage